ooni/ui/cli.py

```diff
@@ -428,7 +463,7 @@ def runWithDaemonDirector(global_options):
     except ImportError:
         print "Pika is required for queue connection."
         print "Install with \"pip install pika\"."
-        sys.exit(7)
+        raise SystemExit(7)
```

Comment:
Actually there is no difference, but I figured it made it a bit easier to scan the code looking for raises rather than looking for a `sys.exit`.